src/model/line/mod.rs

```diff
@@ -26,10 +26,55 @@ impl Line {
             "C4 F3 G3 Bb3 C4 Db4 Eb4 F4 E4 . . .",
             "- Db4 Bb3 Db4 C4 . Bb3 G3 F3 Bb3 F3 Gb3 G3 Gb3 F3 G3 E3 . . .",
             "C4 F3 G3 Bb3 C4 Db4 Bb3 Db4 C4 . .",
+            "G4 Gb4 G4 A4 Bb4 C5 A4 Ab4 G4 C4 C5 .",
```

Comment:
It's pretty arbitrary that these are included in this branch/PR, I'd started writing out some more "inside-the-key" lines at one point but so these are currently unused since for the moment it's not playing any of the inside-the-key lines